server/src/service/stores.ts

```diff
@@ -258,10 +258,3 @@ async function findStoreById(storeId) {
   }
   return mappedStores[0];
 }
-
-module.exports = {
```

Comment:
Any reason to remove this ?

Reply:
replaced with `export` keywords - more typescript-y